firepit/sqlstorage.py

```diff
@@ -960,7 +960,63 @@ def timestamped(
         else:
             res = qry
         return res
+    def getAllNestedObjectsWithAnAttributeOfSCO(
+            self,
+            viewname,
+            path=None,
+            value=None,
+            nameOfColumnTobeAppended='id',
+            limit=None,
+            run=True):
+        
+
+        # Something like this:
+        # select sco."{column}" as "{column}", obs."{ts}" as "{ts}"
+        #   from "{viewname}" sco
+        #     join __contains c on sco.id = c.target_ref
+        #     join "observed-data" obs on c.source_ref = obs.id
+        #   where sco."{column}" = {value};
```

Comment:
This comment block, and indeed the entire function is copied verbatim from the `timestamped` method and one param had it's name changed.  It seems like a better implementation would maybe take the current `timestamped` method and make it a private utility method, and then `timestamped` could just be a wrapper that calls it.  The new method would do the same, with the only difference being the column you wanted to add.